internal/pkg/config/config.go

```diff
@@ -221,8 +221,7 @@ func InitConfig() error {
 		} else {
 			home, err := os.UserHomeDir()
 			if err != nil {
-				fmt.Println(err)
-				os.Exit(1)
+				return
```

Comment:
We should return the error here?

Reply:
No, I can explain.
This code runs inside `once.Do(func() {` which means that we can only run `return`. It also happens in other places in the same function.

We define `err` here and then after we close `once.Do`, we return `err` if its not `nil`.

Reply:
Ahhhh, I see. Thank you!